src/components/footer/footer.scss

```diff
@@ -1,21 +1,23 @@
 @import '~theme/base/variables';
 @import '~theme/base/mediaqueries';
 @import '~theme/base/utils';
+@import '~theme/base/spacing';
 
 .footerMain {
   width: 100%;
 }
 
 .footerInner {
+  composes: PVS;
   display: flex;
   flex-direction: column;
   align-items: center;
   margin-left: 20px;
   margin-right: 20px;
-  padding: 10px 0;
   border-top: 1px solid $color-grey-light;
   
   @include respond-to(tablet) {
+    @include spacing('P', 'V', 'M');
```

Comment:
The reason to do not always use the `@mixin` is because of bundle size, right? If not, we should always use `@mixin`to have a consistency way to use it

Reply:
Nice! So yes. I would use the **composition as default** and then inside `mediaqueries` use the `mixin`